Assets/MixedRealityToolkit/_Core/Devices/UnityInput/GenericUnityController.cs

```diff
@@ -0,0 +1,232 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.Devices;
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
+using Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Devices.UnityInput
+{
+    public class GenericUnityController : BaseController
```

Comment:
Should `BaseController` become `GenericUnityController`? What are some examples of devices that would utilize `BaseController` and not `GenericUnityController`?

Reply:
Not all controllers will be getting input from the Unity Input System, so no.

We already have the Windows Mixed Reality Controllers getting input directly from the Interaction Manager, instead of unity input.